projects.yml

```diff
@@ -586,6 +586,24 @@ clj_xml_validation:
   categories: [XML Schema]
   platforms: [clj]
 
+jinx:
+  name: jinx
+  url: https://github.com/juxt/jinx
+  categories: [Schema]
+  platforms: [clj, cljs]
+
+scjsv:
+  name: scjsv
+  url: https://github.com/metosin/scjsv
+  categories: [Schema]
+  platforms: [clj]
+
+json_schema:
+  name: json-schema
+  url: https://github.com/luposlip/json-schema
+  categories: [Schema]
+  platforms: [clj]
```

Comment:
I've just noticed the existing category `Schema` has items all supporting generic data. On that basis, should these JSON specific libraries be instead in a (new) category `JSON Schema`? I see there is an existing category `XML Schema`

Reply:
Yes, JSON Schema is its own thing, I think. Schema is more for generic data that's already been loaded into Clojure.